components/tools/OmeroPy/src/omero/plugins/upload.py

```diff
@@ -64,9 +70,27 @@ def upload(self, args):
                              " import")
             else:
                 obj = client.upload(file, type=omero_format)
-                self.ctx.out("Uploaded %s as " % file + str(obj.id.val))
+                fileIds.append(obj.id.val)
                 self.ctx.set("last.upload.id", obj.id.val)
 
+        if args.annotate:
+            c = self.ctx.conn(args)
+            update = c.sf.getUpdateService()
+            query = c.sf.getQueryService()
+
+            for fId in fileIds:
+                # obj new FileAnnotation file=OriginalFile:275
+                ann = FileAnnotationI()
+                # f = file get by id
+                ann.file = query.find("OriginalFile", fId)
```

Comment:
Do you need to query the file? Does `ann.file = omero.model.OriginalFileI(fId, False)` work?